core/src/main/scala/kafka/log/remote/RemoteIndexCache.scala

```diff
@@ -38,7 +38,60 @@ import java.util.concurrent.locks.ReentrantReadWriteLock
 object RemoteIndexCache {
   val DirName = "remote-log-index-cache"
   val TmpFileSuffix = ".tmp"
-  val remoteLogIndexCacheCleanerThread = "remote-log-index-cleaner"
+  val RemoteLogIndexCacheCleanerThread = "remote-log-index-cleaner"
+
+  def RemoteLogSegmentIdFromRemoteIndexFileName(fileName: String): Uuid = {
```

Comment:
It looks like we do not use `PascalCase or CamelCase` for methods in object class. Should we continue using `camelCase` here like in other places in this project?

Reply:
fixed in latest commit